pkg/scheduler/metrics/source/metrics_client_elasticsearch.go

```diff
@@ -47,10 +49,16 @@ func NewElasticsearchMetricsClient(address string, conf map[string]string) (*Ela
 		e.indexName = indexName
 	}
 	var err error
+	insecureSkipVerify := conf["tls.insecureSkipVerify"] == "true"
```

Comment:
This way of writing is really concise, but I'm not sure if it's the best choice, can you take a look at it together？ @william-wang @hwdef

Reply:
I think it's OK.

Reply:
I did not find the place to use the `tls.skipVerify` in the code change.